envs/conda_build_config.yaml

```diff
@@ -244,7 +248,8 @@ pytorch_scatter:
 pytorch_sparse:
   - 0.6.10
 pyyaml:
-  - 5.*
+  - 5.*   # [not s390x]
+  - 6.*   # [s390x]
```

Comment:
pyyaml: Can we move back to 5.x on s390x ? We faced some conflicts in CP4D when we moved to tensorflow 2.7.x and we changed pyyaml back to 5.x

Reply:
This change is cherry-picked from 1.5 branch as it is https://github.com/open-ce/open-ce/blob/open-ce-r1.5/envs/conda_build_config.yaml#L240.

Reply:
We were at 6.x level for some time. But faced issues because of some other packages and moved back to 5.x